shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -753,6 +810,46 @@ enum RewriteTableTy {
     Peers,
 }
 
+fn is_prepared_message(message: &mut Message) -> Option<(Vec<u8>, PreparedMetadata)> {
```

Comment:
how about `is_result_prepared_message`.
`is_prepare_message` and `is_prepared_message` are confusingly similar.